### DISCUSSION TITLE
Add a repair for statistics

### DISCUSSION THREAD

**Discussion by erkr** [frenck/spook#857] [Ideas]
Spook adds various useful repairs to HA👍  I would like to have an extra repair when there are statistics to be fixed. Those are only noticed when opening the statistics page in the dev tools

Best Eric

**Comment by frenck** [comment]
In the latest versions of Home Assistant, Home Assistant does that already... 🫣

**Reply by erkr** [reply]
Strange 🧐
I didn’t get a repair when I noticed over 20 items to fix in statistics. That is why I raised this idee. So if it happens again raise a ticket in HA then👌

**Reply by erkr** [reply]
I verified, this doesn't work in core2024.10.4, but I will check later in core2024.11.1 in case that is the latest version you refer to
